### PR TITLE
python3-sh: update to 1.14.3.

### PR DESCRIPTION
**Pull request by pfpulux**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by icp1994** [comment]
The failing test is weird. It always fails for me when run on a clean build but passes on subsequent invocations.

**Comment by classabbyamp** [comment]
I wonder what dir it's listing for this

https://github.com/amoffat/sh/blob/develop/test.py#L862-L866

**Comment by icp1994** [comment]
`masterdir/tmp` is being operated on. Both `ls -A1` and `os.listdir(".")` outputs the contents properly. Even if you manually wipe with `rm -r masterdir/tmp/*`, on a clean build you get `AssertionError: 1 != 0`. But wiping `tmp` and running `./xbps-src check` again passes immediately after that :confounded:

**Comment by classabbyamp** [comment]
maybe another test is creating files in tmp?

**Comment by icp1994** [comment]
It does generate 2 files under `masterdir/tmp` for me after each invocation but that happens in later tests, executed after the two function calls in this test. Even then I couldn't figure out a reason why it would pass the tests on rebuilds with an empty `tmp` since the conditions seem identical to a clean build. Might just be that test has some race condition or flaky or [weird issue with ls and wc](https://unix.stackexchange.com/a/113472).

**Comment by classabbyamp** [comment]
ok, let's disable this test then.

File: srcpkgs/python3-sh/template [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-checkdepends="python3-Pygments python3-coverage python3-coveralls python3-docopt python3-docutils flake8"
+# checkdepends="python3-Pygments python3-coverage python3-coveralls python3-docopt python3-docutils flake8"
 short_desc="Python subprocess replacement (Python3)"
 maintainer="pulux <pulux@pf4sh.de>"
 license="MIT"
 homepage="https://github.com/amoffat/sh"
 distfiles="${PYPI_SITE}/s/sh/sh-${version}.tar.gz"
-checksum=9d7bd0334d494b2a4609fe521b2107438cdb21c0e469ffeeb191489883d6fe0d
+checksum=e4045b6c732d9ce75d571c79f5ac2234edd9ae4f5fa9d59b09705082bdca18c7
+make_check=no
```
Comment (classabbyamp) [inline srcpkgs/python3-sh/template:16]: I don't think we need to disable test altogether if just one fails

**Pull request closed without merging**